packages/components/psammead-inline-link/package-lock.json

```diff
@@ -34,9 +34,9 @@
       }
     },
     "@bbc/gel-foundations": {
-      "version": "1.2.0",
-      "resolved": "https://registry.npmjs.org/@bbc/gel-foundations/-/gel-foundations-1.2.0.tgz",
-      "integrity": "sha512-OpEJf42FSgyZRN0e9pJFm9eYmzbhfB7W7EWlrjh46fWlozyVwKA4uaTPUoGHLbXJkMOhPdpfM3gOAxbuM0FnrQ==",
+      "version": "3.0.0",
```

Comment:
good spot. I did a search and replace without looking at the file name. Will remove

Reply:
@DenisBBC so it turns out `gel-foundations` is supposed to be there because it's used by `storybook-helpers` which we use in `inline-link`. Removing it broke stuff 🙈

Reply:
Ok, fair enough 👍🏻